lib/persist_campaign/controller.rb

```diff
@@ -8,10 +8,22 @@ def self.included base
     module InstanceMethods
       # Solution provided by ndp's save-the-campaign gem (http://github.com/ndp/save-the-campaign),
       # reproduced here under the MIT license
-      def _compute_redirect_to_location_with_persist_campaign_params(options = {})
+
+      if Rails.version >= "4.2.1"
```

Comment:
I *think* this is probably safe, but I wonder if it's even safer to use `Gem::Version` instances here (http://stackoverflow.com/questions/2051229/how-to-compare-versions-in-ruby)